generate-release/src/main.rs

```diff
@@ -59,10 +70,20 @@ enum Commands {
         #[arg(short, long)]
         weight: i32,
 
+        /// Release version i.e.: '0.13', '0.14', etc.
+        #[arg(short, long)]
+        release_version: String,
+
         /// Path used to output the generated file. Defaults to ./migration-guide.md
         #[arg(short, long)]
-        path: Option<std::path::PathBuf>,
+        output_path: Option<std::path::PathBuf>,
     },
+    /// Release notes:
+    /// * Gets all merged PRs
+    /// * Collect each author of closed PRs (Should this just list all contributors?)
```

Comment:
```suggestion
    /// * Collect each author of merged PRs
```
We want to be accurate for "contributors for this release"

Reply:
I think I added that comment because this doesn't account for reviewers and I really want reviewers to be mentioned in things like that, but yeah, right now it uses the merged PRs only.